plugins/package-managers/go/src/funTest/assets/projects/synthetic/gomod/go-as-dep-with-transitive-deps-expected-output.yml

```diff
@@ -1,379 +1,23 @@
 ---
 project:
-  id: "GoMod::my-module:<REPLACE_REVISION>"
-  definition_file_path: "<REPLACE_DEFINITION_FILE_PATH>"
+  id: "GoMod::my-module:888c785f7811c9a42aaf6a84cd78e57b4157d6b7"
+  definition_file_path: "plugins/package-managers/go/src/funTest/assets/projects/synthetic/gomod/go-as-dep-with-transitive-deps/go.mod"
   declared_licenses: []
   declared_licenses_processed: {}
   vcs:
     type: "Git"
-    url: "<REPLACE_URL_PROCESSED>"
-    revision: "<REPLACE_REVISION>"
-    path: "<REPLACE_PATH>"
+    url: "ssh://git@github.com/oss-review-toolkit/ort.git"
+    revision: "888c785f7811c9a42aaf6a84cd78e57b4157d6b7"
+    path: "plugins/package-managers/go/src/funTest/assets/projects/synthetic/gomod/go-as-dep-with-transitive-deps"
   vcs_processed:
     type: "Git"
-    url: "<REPLACE_URL_PROCESSED>"
-    revision: "<REPLACE_REVISION>"
-    path: "<REPLACE_PATH>"
+    url: "ssh://git@github.com/oss-review-toolkit/ort.git"
+    revision: "888c785f7811c9a42aaf6a84cd78e57b4157d6b7"
+    path: "plugins/package-managers/go/src/funTest/assets/projects/synthetic/gomod/go-as-dep-with-transitive-deps"
```

Comment:
The placeholders should be maintained here.

Reply:
Oh, sorry. I missed to review this file. I'll turn it into a draft for now and try to come back to it tomorrow.